system/jhtree/jhtree.cpp

```diff
@@ -2689,7 +2737,9 @@ const CJHTreeNode *CNodeCache::getNode(const INodeLoader *keyIndex, unsigned iD,
                 ownedCacheEntry->noteReady(node);
             }
             else
-                (*dupMetric[cacheType])++;
+            {
+                curCache.numDups->fastInc(1);
```

Comment:
should this also be adding to the ctx? e.g. ctx->noteStatistic(dupStatId[cacheType], 1);

Reply:
I don't think the dups stat is relevant to the context (it's only really relevant globally) - ditto evictions